src/python/pants/backend/python/lint/bandit/rules_integration_test.py

```diff
@@ -187,6 +188,9 @@ def test_skip(rule_runner: RuleRunner) -> None:
     assert not result
 
 
+@pytest.mark.skipif(
+    not (has_python_version("3.6") or has_python_version("3.7")), reason="Missing requisite Python"
```

Comment:
Why this?

Reply:
From line 200 `interpreter_constraints=['>=3.6,<3.8']`